iothub/device/samples/PnpDeviceSamples/TemperatureController/ComponentTemperatureControllerSampleOld.cs

```diff
@@ -86,39 +87,40 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
             };
 
             await _deviceClient.SendEventAsync(message, cancellationToken);
-            _logger.LogDebug($"Telemetry: Sent - {JsonConvert.SerializeObject(telemtry)} in KB.");
+            _logger.LogDebug($"Telemetry: Sent - {System.Text.Json.JsonSerializer.Serialize(telemetry)}.");
 
             // Subscribe and respond to event for writable property "humidityRange" under component "thermostat1".
             await _deviceClient.SetDesiredPropertyUpdateCallbackAsync(async (desired, userContext) =>
             {
                 string propertyName = "humidityRange";
-                if (!desired.Contains(Thermostat1) || !((JObject)desired[Thermostat1]).ContainsKey(propertyName))
+                if (!desired.Contains(Thermostat1) || !((JObject)desired[Thermostat1]).TryGetValue(propertyName, out JToken humidityRangeToken))
                 {
                     _logger.LogDebug($"Property: Update - Received a property update which is not implemented.\n{desired.ToJson()}");
                     return;
                 }
 
-                HumidityRange targetHumidityRange = ((JObject)desired[Thermostat1]).Value<HumidityRange>(propertyName);
+                HumidityRange targetHumidityRange = System.Text.Json.JsonSerializer.Deserialize<HumidityRange>(humidityRangeToken.ToString());
 
-                var propertyPatch = new TwinCollection();
-                var componentPatch = new TwinCollection()
+                var propertyPatch = new Dictionary<string, object>();
+                var componentPatch = new Dictionary<string, object>
                 {
                     ["__t"] = "c"
                 };
-                var temperatureUpdateResponse = new TwinCollection
+                var temperatureUpdateResponse = new
```

Comment:
Same as here. I opted to not use an anonymous class since we had specified TwinCollection